tests/functional/test_s3.py

```diff
@@ -1340,6 +1340,46 @@ def test_500_error_with_non_xml_body(self):
             self.assertEqual(len(http_stubber.requests), 2)
 
 
+class TestS3Parser(BaseS3OperationTest):
+    def test_valid_expires_value_in_response(self):
+        expires_value = "Thu, 01 Jan 1970 00:00:00 GMT"
+        mock_headers = {'expires': expires_value}
+        s3 = self.session.create_client("s3")
+        with ClientHTTPStubber(s3) as http_stubber:
+            http_stubber.add_response(headers=mock_headers)
+            response = s3.get_object(Bucket='mybucket', Key='mykey')
+            self.assertIn('Expires', response)
+            self.assertIsInstance(response['Expires'], datetime.datetime)
+            self.assertIn('ExpiresString', response)
+            self.assertIsInstance(response['ExpiresString'], str)
+            self.assertEqual(response['ExpiresString'], expires_value)
```

Comment:
An extra assertion isn't blocking, but the `isInstance` calls are redundant with the `assertEqual` calls.  - this goes for many of these tests

Reply:
You're right. I removed those.